Sections/EEA/GPPExtension: IAB Europe TCF.md

```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
       Note: With TCF v2.2 support for legitimate interest for purpose 3 to 6 has been deprecated. Bits 2 to 5 are required to be set to <code>0</code>.</td>
     </tr>
     <tr>
+  <tr style="background-color:#000;color:#FFF;">
+    <td colspan="4">
+        <strong>Specific Jurisdiction Disclosures</strong>
+    </td>
+  </tr>
   <td>PurposeOneTreatment</td>
   <td>Boolean</td>
-  <td>1 Purpose 1 was NOT disclosed at all.<br><br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br><br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
+  <td>1 Purpose 1 was NOT disclosed at all.<br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
   </tr>
   <tr>
   <td>PublisherCC</td>
   <td>String (2)</td>
-  <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
-    Each letter is encoded as 6 bits, <code>a=0..z=25</code>. <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2"> ISO 3166-1 alpha-2 code </a> </td>
+    <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
```

Comment:
just for sake of clarity lets add "lowercase" to it

Reply:
Declined by FSWG